src/ansys/fluent/core/__init__.py

```diff
@@ -84,6 +84,9 @@ def version_info() -> str:
 # Whether stream and cache commands state
 DATAMODEL_USE_NOCOMMANDS_DIFF_STATE = True
 
+# Whether to use remote gRPC file transfer service
+REMOTE_GRPC_FILE_TRANSFER_SERVICE = None
```

Comment:
It is simpler if a boolean config is either True or False.

Reply:
Done. Thank you.

Reply:
Done. Thank you.